module/Activity/src/Controller/AdminCategoryController.php

```diff
@@ -29,8 +29,9 @@ class AdminCategoryController extends AbstractActionController
      */
     public function __construct(
         ActivityCategoryService $categoryService,
-        Translator $translator
-    ) {
+        Translator              $translator
+    )
+    {
```

Comment:
This is not PSR-12 compliant, the `)` and `{` should be on the same line and I would add a `,` after `$translator` (and personally I wouldn't indent `$translator` to the same level as `$categoryService`).

Reply:
Not sure how this happened....